spec/status.yml

```diff
@@ -233,3 +233,74 @@ components:
               application/json:
                 schema:
                   $ref: '#/components/schemas/AuthenticatorsResponse'
+
+    Health:
+      get:
+        x-conjur-settings:
+          dap-only: true
+        tags:
+        - "status"
+        summary: "Health info about conjur"
+        description: |
+          "You can request health checks against any cluster node using the Conjur API.
+          These routes do not require authentication.
+
+          The health check attempts an internal HTTP or TCP connection to
+          each DAP service. It also attempts a simple transaction against all internal databases."
+        operationId: "health"
+        responses:
+          "200":
+            description: The tests were successful
+            content:
+              application/json:
+                schema:
+                  type: object
+          "502":
+            description: The tests failed
+
+    RemoteHealth:
+      get:
+        x-conjur-settings:
+          dap-only: true
+        tags:
+          - "status"
+        summary: "Health info about a given DAP server"
+        description: |
+          "Use the remote_health route to check the health of any DAP Server from any other DAP Server.
+          With this route, you can check master health relative to a follower, or follower health relative
+          to a standby, and so on."
+        operationId: "remoteHealth"
+        parameters:
+          - name: remote
+            in: path
+            schema:
+              type: string
+            required: true
+            description: "The hostname of the remote to check"
+            example: "conjur.myorg.com"
+        responses:
+          "200":
+            description: The tests were successful
+            content:
+              application/json:
+                schema:
+                  type: object
+          "502":
+            description: The tests failed
+    Info:
+      get:
+        x-conjur-settings:
+          dap-only: true
+        tags:
+          - "status"
+        summary: "Basic information about the DAP server"
+        description: |
+          "Information about authenticator services in the Conjur configuration"
```

Comment:
This endpoint isn't just about authenticator services - we should have a more detailed description here. You can see info about this endpoint [here](https://github.com/cyberark/dap-wiki/blob/72c888aa4907ee91fc2ca359d4c443f4daebe144/reference/endpoints/info.md)

Reply:
I updated the description with some more detail and expanded the response objects schema definition. Between those two changes it should better encompass the endpoint.